app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -582,6 +587,25 @@ object TelemetryWrapper {
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.MENU, Value.RELOAD).queue();
     }
 
+    @JvmStatic
+    fun SSLError(fromPage: Boolean, error: SslError) {
```

Comment:
detekt (on taskcluster) is throwing a warning:
> FunctionNaming - [SSLError] at focus-android/app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt:590:5

Probably because it doesn't start with a lowercase character. Maybe sslErrorEvent(...)?